store/project_member.go

```diff
@@ -123,6 +123,79 @@ func (s *ProjectMemberService) DeleteProjectMember(ctx context.Context, delete *
 	return nil
 }
 
+// SetProjectMember set the project member with provided project member list
+func (s *ProjectMemberService) SetProjectMember(ctx context.Context, projectID int, operatorID int, setList []*api.ProjectMemberCreate) (createdMember, deletedMember []*api.ProjectMember, err error) {
```

Comment:
This should be called BatchPatchProjectMemberList and create separate struct to include projectID int, operatorID int.

type ProjectMemberPatchBatch struct {
	ID int

	// Standard fields
	// Value is assigned from the jwt subject field passed by the client.
	UpdaterID int

       list []*api.ProjectMemberCreate
}

Reply:
I think BatchPatch is inappropriate, since we may delete or create new record.